csv_template_ingester/tests/test_template_csv_parser.py

```diff
@@ -23,6 +23,12 @@ def test_get_keyword():
     assert keyword_two == 'NAME'
     assert column_header_two == ''
 
+def test_get_keyword_colon_count():
+    try:
```

Comment:
please add code comments for what this test is checking for.

Reply:
I added the comments and followed some better practices suggested by Hutch. If you have any more suggestions, let me know!